### ISSUE TITLE
Fatal error using the latest version of the phar archive (1.4.0) 

### ISSUE THREAD

**Issue by asimlqt** [doctrine/migrations#444]
Seems like the Ocramius dependency is missing from the 1.4.0 phar archive!

**Comment by chrisguitarguy** [comment]
I think this might be fixed in #439

**Comment by mikeSimonson** [comment]
@asimlqt Can you post how you triggered the fatal error ?

**Comment by asimlqt** [comment]
@chrisguitarguy Thanks, the issue had indeed been resolved in that commit.

@mikeSimonson Downloaded the phar from the releases page, executed 'migrations:generate' to create a new migration which worked fine and then tried to execute it using 'migrations:migrate', that triggered the fatal error.

However the issue has been fixed in the source code but it would be useful if the broken phar on the releases page be replaced with a new working version. I have created one and can send it to someone with permissions to update because it's not as straight forward generating a phar file manually. I came across a couple of issues:

1. I had to delete the vendor folder and re-install the dependencies with '--no-dev'. Without this, it will also throws a fatal error. Complains about missing 'hamcrest' dependency.
2.  When generating the phar archive I was getting the 'too many open files' error so I had to update the open files soft limit. I got this error on both, osx and linux.
